framework/docs/src/0_abstract_documentation.md

```diff
@@ -0,0 +1,60 @@
+# Abstract Documentation
+
+Hi and welcome to Abstract, the interchain CosmWasm development platform! Abstract's CosmWasm framework is your gateway 
+to taking amazing ideas from concept to reality. Whether you're a coding genius or just getting started, Abstract has got your back! By building composable modules you'll be crafting scalable masterpieces in no time.
+
+## Getting Started
+
+You can read the documentation in the order it is presented, or you can jump to the section that interests you the most! We 
+also have a section with code tutorials that will help you get started with the Abstract SDK if you are more of a hands-on.
+
+- [Abstract Overview](./1_abstract_platform_overview.md)
+- [Tutorials](./10_tutorials.md)
+
+## Explore the Abstract SDK
+
+- [Abstract Overview](./1_abstract_platform_overview.md)
+- [Core Features](./1_abstract_platform_overview.md)
+- [Platform Essentials](./1_abstract_platform_overview.md)
+- [Build with Abstract](./4_get_started/)
+- [Glossary](./9_glossary.md)
+
+## Use Cases
+
+How can you use Abstract? The limit is your imagination, we can't wait to see what you build! Here are some ideas to get
+you started:
+
+- Yield farming platforms
+- Decentralized exchanges
+- Lending protocols
+- Prediction markets
+- NFT marketplaces
+- Cross-chain bridges
+- Supply chain managements
+- Insurance and underwriting
+- DAOs
+- Interchain protocols
+- And more!
+
+See how others are using Abstract in the [Use Cases](./7_use_cases/index.md) section.
+
+## Help and Support
+
+If you feel lost or have any doubts along the way, please reach out to us! We are here to help you!
+
+- [Discord](https://discord.com/invite/uch3Tq3aym)
+- [X](https://x.com/AbstractSDK)
+
+## Want to make Abstract better?
+
+We are always looking for ways to improve Abstract and welcome everybody to contribute to the project. Look at the 
+[Contributing & Community](./contributing.md) section if you want to get involved.
+
+## Links
+
+- [Website](https://abstract.money/)
+- [Github](https://github.com/AbstractSDK)
+- [Medium](https://medium.com/@abstractmoney)
+- [Docs](https://docs.abstract.money/)
+- [Account Console](https://app.abstract.money)
```

Comment:
What do you feel about `https://console.abstract.money`? Would that be understandable for devs that it's for them not the "end-users"?

Reply:
That is definitely better! I was thinking the `app` subdomain was a bit strange actually, let's change it?